src/Shared/E2ETesting/BrowserFixture.cs

```diff
@@ -132,18 +132,16 @@ private async Task<(IWebDriver browser, ILogs log)> CreateBrowserAsync(string co
 
                     return (driver, logs);
                 }
-                catch
+                catch (Exception ex)
                 {
-                    if (attempt >= maxAttempts)
-                    {
-                        throw new InvalidOperationException("Couldn't create a Selenium remote driver client. The server is irresponsive");
-                    }
+                    output.WriteLine($"Error initializing RemoteWebDriver: {ex.Message}");
                 }
+
                 attempt++;
+
             } while (attempt < maxAttempts);
 
-            // We will never get here. Keeping the compiler happy.
-            throw new InvalidOperationException("Couldn't create a Selenium remote driver client. The server is unresponsive");
+            throw new InvalidOperationException("Couldn't create a Selenium remote driver client. The server is irresponsive");
```

Comment:
```suggestion
            throw new InvalidOperationException("Couldn't create a Selenium remote driver client. The server is anti-responsive");
```

(Joking, but I don't actually understand this change)

Reply:
Completely orthogonal comment, I've seen selenium timeout in some tests, I'm wondering if we have instances where it becomes irresponsible. And if we should change the code that initializes selenium (in the future) to do this check on a per-test basis and switch the server (start a new one) if it detects the current one is not being responsive.